bakery_lint/fonttests/test_ttf.py

```diff
@@ -402,20 +407,25 @@ def test_check_full_font_name_begins_with_family_name(self):
         for entry in font.names:
             if entry.nameID != 1:
                 continue
+            familyname = entry
             for entry2 in font.names:
                 if entry2.nameID != 4:
                     continue
-                if (entry.platformID == entry2.platformID
-                        and entry.platEncID == entry2.platEncID
-                        and entry.langID == entry2.langID):
-
-                    entry2value = Font.bin2unistring(entry2)
-                    entryvalue = Font.bin2unistring(entry)
-                    if not entry2value.startswith(entryvalue):
-                        _ = ('Full font name does not begin with family'
+                fullfontname = entry2
+
+                #FIX-ME: I think we should still compare entries
```

Comment:
Yes, it seems Glyphs handles these tables differently; I didn't yet find out why, asked in https://forum.glyphsapp.com/t/name-table-differences-between-macos-and-windows/3904